app/components/delete-meeting.js

```diff
@@ -15,9 +15,23 @@ export default class DeleteMeetingComponent extends Component {
   }
   @task
   *deleteMeeting() {
+    const meetingId = this.args.meeting.id;
     yield this.args.meeting.destroyRecord();
     this.displayDeleteModal = false;
-    this.router.transitionTo('inbox.meetings');
+    setTimeout(() => {
+      const pollingInterval = setInterval(async () => {
+        try {
+          const response = await fetch(`/zittingen/${meetingId}`);
```

Comment:
So `findRecord` fails because the store already knows the record is deleted and refuses to query again.
But could you try `this.store.query` here? It's more in line with the rest of the app and might make maintenance easier down the line (since it'll be model-aware)

Reply:
```this.store.query``` does not seem to work as I cannot correctly pass the meeting id to this method. Same issue with the ```queryRecord``` method.

Reply:
Eventually I settled on using `fetch` as it was faster. Additionally `this.store.query` contained some issues while resolving it when the server returned a 500 status code.